packages/cashier/src/pages/account-transfer/account-transfer-form/account-transfer-form.tsx

```diff
@@ -184,49 +180,68 @@ const AccountTransferForm = observer(
         React.useEffect(() => {
             accounts_from = [];
             mt_accounts_from = [];
-            dxtrade_accounts_from = [];
-            derivez_accounts_from = [];
+            other_cfds_accounts_from = [];
             accounts_to = [];
             mt_accounts_to = [];
-            dxtrade_accounts_to = [];
-            derivez_accounts_to = [];
+            other_cfds_accounts_to = [];
 
             accounts_list.forEach((account, idx) => {
                 const text = <AccountOption idx={idx} account={account} />;
                 const value = account.value;
 
+                const should_disable_other_cfds_selected_from =
+                    selected_to.is_mt && (account.is_dxtrade || account.is_derivez);
+                const should_disable_mt_selected_from =
+                    (selected_to.is_dxtrade || selected_to.is_derivez) && account.is_mt;
+                const should_disable_other_mt_accounts_selected_from = selected_from.is_mt && account.is_mt;
+                const should_disable_dxtrade_selected_from = selected_from.is_dxtrade && account.is_dxtrade;
+                const should_disable_derivez_selected_from = selected_from.is_derivez && account.is_derivez;
+
+                const selected_from_is_disabled =
+                    should_disable_other_cfds_selected_from ||
+                    should_disable_mt_selected_from ||
+                    should_disable_other_mt_accounts_selected_from ||
+                    should_disable_dxtrade_selected_from ||
+                    should_disable_derivez_selected_from;
+
```

Comment:
```suggestion
                const should_disable_other_cfds_selected_from =
                    selected_to.is_mt && (account.is_dxtrade || account.is_derivez);
                const should_disable_mt_selected_from =
                    (selected_to.is_dxtrade || selected_to.is_derivez) && account.is_mt;
                const should_disable_other_mt_accounts_selected_from = selected_from.is_mt && account.is_mt;
                const should_disable_dxtrade_selected_from = selected_from.is_dxtrade && account.is_dxtrade;
                const should_disable_derivez_selected_from = selected_from.is_derivez && account.is_derivez;

                const selected_from_is_disabled =
                    should_disable_other_cfds_selected_from ||
                    should_disable_mt_selected_from ||
                    should_disable_other_mt_accounts_selected_from ||
                    should_disable_dxtrade_selected_from ||
                    should_disable_derivez_selected_from;

```

this code is being duplicated in line#223 to uptil line#236. Can we remove the duplicated code if possible?